app/src/main/java/io/intelehealth/client/activities/privacyNoticeActivity/PrivacyNotice_Activity.java

```diff
@@ -61,8 +64,42 @@ protected void onCreate(Bundle savedInstanceState) {
                 obj = new JSONObject(String.valueOf(FileUtils.encodeJSON(this, AppConstants.CONFIG_FILE_NAME)));
                 }
 
-            String privacy_string = obj.getString("privacyNoticeText");
-            privacy_textview.setText(privacy_string);
+            SharedPreferences sharedPreferences = getSharedPreferences("CommonPrefs", Activity.MODE_PRIVATE);
+            if(sharedPreferences.getAll().values().contains("cb"))

```

Comment:
don't try to check all the keys in shared pref. 
we can check with a single key (lang). No need to check all the key.